sdk/src/wallet/account/operations/output_claiming.rs

```diff
@@ -294,15 +294,10 @@ where
             Some(new_native_tokens.clone().finish()?)
         };
 
-        // Check if the new amount is enough for the storage deposit, otherwise increase it to this
-        let mut required_amount = if possible_additional_inputs.is_empty() {
-            required_amount_for_nfts
-        } else {
-            required_amount_for_nfts
-                + BasicOutputBuilder::new_with_amount(Output::AMOUNT_MIN)
-                    .with_native_tokens(option_native_token.into_iter().flatten())
-                    .rent_cost(rent_structure)
-        };
+        // Get the required amount using a sufficient storage deposit
+        let mut required_amount = BasicOutputBuilder::new_with_amount(required_amount_for_nfts)
+            .with_native_tokens(option_native_token.into_iter().flatten())
+            .min_storage_deposit_amount(rent_structure, token_supply)?;
```

Comment:
This change doesn't seem right, before depending on `if possible_additional_inputs.is_empty() {` it did something different

Reply:
Yeah, but I'm pretty sure that was wrong

Reply:
I don't think it was wrong, it was for the case when there are only NFTs to claim and no other inputs are available
If you add the min_storage_deposit_amount() for a basic output there, you make this amount additional required even though it's not